core/src/test/java/io/questdb/griffin/SqlParserTest.java

```diff
@@ -6035,10 +6110,11 @@ public void testSelectSumFromSubQueryLimit() throws SqlException {
     }
 
```

Comment:
Wouldn't it be better to fix aliases in this test and optionally create another one that'd test duplicate column error ?

Reply:
I can't find a fixed version of 'testSelectSumSquared' with unique aliases .